flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/source/enumerator/TestContinuousSplitPlannerImpl.java

```diff
@@ -173,50 +190,10 @@ public void testTableScanThenIncrementalWithNonEmptyTable() throws Exception {
     }
   }
 
-  @Test
+  @TestTemplate
```

Comment:
why not adding the args directly here?
```
@ParameterizedTest
@EnumSource(value = StreamingStartingStrategy.class, names = { "INCREMENTAL_FROM_LATEST_SNAPSHOT", "INCREMENTAL_FROM_LATEST_SNAPSHOT_EXCLUSIVE" })
```

Reply:
@rodmeneses asked `@TestTemplate` hence I tried that, I thought this was some sort of convention in Iceberg, changed now to `@ParameterizedTest`